dali/image/png.cc

```diff
@@ -47,8 +53,8 @@ DALIError_t GetPNGImageDims(const uint8 *png, int size, int *h, int *w) {
   // 1 byte : Compression method
   // 1 byte : Filter method
   // 1 byte : Interlace method
-  *w = ReadIntFromPNG(IHDR + 8);
-  *h = ReadIntFromPNG(IHDR + 12);
+  *w = ReadIntFromPNG(png_size + 8);
+  *h = ReadIntFromPNG(png_size + 12);
```

Comment:
Bounds checking?
Also maybe it's just me but `png_size` is a confusing name for what I think this is actually pointing to?

Reply:
Done